### ISSUE TITLE
[BUG]: sorting is messed up

### ISSUE THREAD

**Issue by TheOne299** [yuliskov/SmartTube#4077]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/yuliskov/SmartTube/issues) or [closed](https://github.com/yuliskov/SmartTube/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://github.com/yuliskov/SmartTube#faq) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.

### Affected version

24.53

### Device Type

Smart TV/Box

### Affected Android

Android TV 11

### Steps to reproduce the bug

Go to channel list

### Actual behavior

Until now, channels with new content were coloured red and in alphabetical order and the other channels were below that in alphabetical order. Now there is just the separation between channels with or without new content. No alphabetical order within. 

I changed nothing before or after the update to my settings. 

### Additional information

![GridArt_20241108_113554409](https://github.com/user-attachments/assets/373bbf1f-f2e1-44ab-988e-b2751d242bf2)

**Comment by DiscoFever78** [comment]
+1  same issue here

**Comment by Fat-Zilla** [comment]
Can confirm this is broken in last 2 builds. This is specific to Alphabetical. Work around right now is to change your sorting to Alphabetical (fast previews) until this is fixed.

Settings > User Interface > Channels Section Sorting

**Comment by dontwantonebutneedone** [comment]
Same problem here. And some content still isn't showing in channel view unless you explicitly "open channel".

We seem to keep going backwards to old, resolved problems.

**Comment by CaffeineJunkie** [comment]
Yeah, sorting of channels is messed up now. Also after exiting a video of a channel in the red new content section the channel is now not red and sorted with the rest. I have to leave channels and come back to see the rest or scroll all the way back.

**Comment by ACR-Jeff** [comment]
I am facing the same issue on both stable and beta, I haven't updated my stable version in months because it just worked without issues, I was using the beta version today and noticed this issue, So I thought I would use the stable version with out the issue, Nope the issue is present in the stable version as well. Which makes me believe it's on YT's side or something they've changed.

**Comment by ACR-Jeff** [comment]
Duplicate https://github.com/yuliskov/SmartTube/issues/4070
